src/test/java/bio/terra/service/policy/flight/CreateSnapshotPolicyStepTest.java

```diff
@@ -75,4 +76,20 @@ void testNoPolicyDoUndo() throws Exception {
     assertThat(undoResult.getStepStatus(), equalTo(StepStatus.STEP_RESULT_SUCCESS));
     verify(policyService, never()).deletePao(SNAPSHOT_ID);
   }
+
+  @Test
+  void testCreatePolicyServiceNotEnabled() throws Exception {
+    mockFlightMap();
+    CreateSnapshotPolicyStep step = new CreateSnapshotPolicyStep(policyService, true);
+    var exception = new FeatureNotImplementedException("Policy service is not enabled");
+    doThrow(exception).when(policyService).createSnapshotPao(SNAPSHOT_ID, policies);
```

Comment:
If this exception is thrown on `createSnapshotPao`, I think it would also be thrown on `deletePao`.

Reply:
It does! The exception gets thrown by `policyServiceConfiguration.tpsEnabledCheck()` in both `createPao` and `deletePao`: https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/java/bio/terra/service/policy/PolicyService.java#L70